membership_variable_period/__openerp__.py

```diff
@@ -1,13 +1,15 @@
 # -*- coding: utf-8 -*-
 # (c) 2015 Antiun Ingeniería S.L. - Pedro M. Baeza
+# (c) 2016 Camptocamp SA - Denis Leemann
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 {
     'name': 'Variable period for memberships',
-    'version': '8.0.1.0.0',
+    'version': '9.0.1.0.0',
     'license': 'AGPL-3',
     'category': 'Association',
     'author': 'Antiun Ingeniería S.L., '
               'Serv. Tecnol. Avanzados - Pedro M. Baeza, '
```

Comment:
Can you please replace these 2 companies by the new one we have founded: `Tecnativa`?